spec/models/address_search_spec.rb

```
@@ -24,6 +24,10 @@
       params = { postcode: 'n16 8qr' }
       expect(AddressSearch.new(params).data).to eq(postcode: 'N16 8QR')
     end
+
+    it 'returns the search data even if no params were provided' do
```

I can't work out why this is - I've checked and nothing seems to break if it's not true.

Is my understanding correct?: if no params are provided then the form is invalid, and if the form is invalid then the caller shouldn't be asking it for data.

If that's the case then would it be better to raise an exception in this scenario?

---

Was removed